app/src/main/res/layout-sw600dp-port/profile_chooser_fragment.xml

```diff
@@ -78,14 +78,15 @@
           android:paddingBottom="@{hasProfileEverBeenAddedValue ? @dimen/profile_chooser_padding_bottom_profile_already_added : @dimen/bottom_white_space}"
           android:requiresFadingEdge="vertical"
           android:scrollbars="none"
+          android:tag="profile_recycler_view"
```

Comment:
Why is this needed?

Reply:
We are using Tag in the `ProfileChooserSpanTest` to get the RecyclerView and use the span in the GridLayoutManager. 
https://github.com/oppia/oppia-android/blob/8e4b43938a5f07e51124fefdec4e39385922ac47/app/src/test/java/org/oppia/android/app/testing/ProfileChooserSpanTest.kt#L323

Reply:
Resolving on behalf of Ben